catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -309,6 +309,8 @@
         <property name="productCacheDirectory" value=""/>
         <property name="context" ref="blueprintBundleContext"/>
         <property name="xmlConfigFilename" value="reliableResource-hazelcast.xml"/>
+        <!--            TODO: Temporary injection until removal            -->
```

Comment:
Thanks for the explanation @oconnormi. I was trying to make sense of the comment. The `removal` part is for the old interface? @Lambeaux can you clarify the comment?